packages/buidler-core/src/internal/buidler-evm/provider/output.ts

```diff
@@ -187,9 +187,7 @@ export function getRpcTransactionReceipt(
     gasUsed: numberToRpcQuantity(new BN(receipt.gasUsed)),
     contractAddress:
       createdAddress !== undefined ? bufferToRpcData(createdAddress) : null,
-    logs: receipt.logs.map((log, logIndex) =>
-      getRpcLog(log, tx, block, index, logIndex)
-    ),
+    logs: receipt.logs,
```

Comment:
I think I wouldn't remove, even if these types match. `ethereumjs-vm` type this as any, and may change them in the future, making it hard for us to detect any incompatibility introduced.

Reply:
I mean, changing these `any`-typed values would also break, but that would be more self-contained in `getRpcLog`.

Reply:
Got it, in that case, I would suggest we create our own `TxReciept` interface where logs are an array of `RpcLogOutput` and use that one in `TxBlockResult` interface. That way we a single point of conversion.